calico/felix/config.py

```diff
@@ -217,4 +218,3 @@ def validate_addr(self, name, addr):
             raise ConfigException("Invalid or unresolvable %s value : %s" %
                                   (name, addr),
                                   self._config_path)
```

Comment:
Please remove this file from the change - we don't want the double blank line or to remove the blank line at end of file.